Detectors/MUON/MCH/Calibration/test/run-tfbuilder.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+TRANSPORT=shmem
+#TRANSPORT=zeromq
+
+StfBuilder --id stf_builder-0 --transport $TRANSPORT --detector TPC --dpl-channel-name=dpl-chan --channel-config "name=dpl-chan,type=push,method=bind,address=ipc://@tf-builder-pipe-0,transport=shmem,rateLogging=1" --data-source-dir="$1" --data-source-enable --control=static #--max-built-stfs 10
```

Comment:
Ciao @aferrero2707 , 

You have `TPC`, but it should be `MCH`...

Chiara

Reply:
and I think you should have:
`transport=$TRANSPORT`
(even though this does not really matter)